src/logsearch-config/Rakefile

```diff
@@ -32,7 +33,7 @@ end
 
 desc "Runs JSON validation tests"
 task :json, [:json_files] => :build do |t, args|
-  args.with_defaults(:json_files => "$(find target -name *.json)")
+  args.with_defaults(:json_files => "$(find target -name *.json ! -name kibana4-dashboards.json)")
```

Comment:
@jmcarp Why do you skip dashboard jsons from the test? I think it makes sense to test these files too.